corehq/apps/data_cleaning/tasks.py

```diff
@@ -65,6 +61,15 @@ def commit_data_cleaning(self, bulk_edit_session_id):
     return form_ids
 
 
+@retry_on_integrity_error(max_retries=3, delay=0.1)
```

Comment:
I'm not sure we need this retry decorator

Reply:
removed this any other retries